build/ci/delete_unused_disks.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+import os
+import json
+
+project = os.environ['GCLOUD_PROJECT']
+
+os.system('gcloud compute disks list --filter="-users:*" --format="json" --project {} > unused_disks.json'
+          .format(project))
+
+with open('unused_disks.json', 'r') as f:
+    content = f.read()
+    try:
+        parsed_json_dict = json.loads(content)
+        if len(parsed_json_dict) == 0:
+            print("There is no unused disks. Congratulations!")
+        else:
+            for entry in parsed_json_dict:
+                name = entry['name']
+                head, tail = os.path.split(entry['zone'])
+                os.system('gcloud compute disks delete {} --project {} --zone {} --quiet'
```

Comment:
I'm a bit more familiar with using the `subprocess` Python module. Does `os.System` display stdout if the command fails?

Reply:
Output will be similar to what you can see in command line. At least I tried different commands and saw the same output as in command line.